PlayBooks/Official/Version-1/Large-Company/en/OpenChain Playbook - Large Company.md

```diff
@@ -0,0 +1,192 @@
+
+### The Legal Expert
+
+He has been working in the field of software licensing for eight years and has always worked at this company. He is experienced in other areas of law too, particularly related to trademarks. He has limited experience in open source licensing and has relied on the engineering department to understand market norms. He has spent some time reading open source licenses, interpreting them and seeking to reconcile his interpretation with online resources. The match has not always been perfect.
+
+### The Management Representative
+
+They have been with this company for four years. Previously they were with a competing company for seven years. They have deep experience in product management but have largely relied on members of their teams or departments to deal with the details of software development and legal matters. They are aware that open source process management may need changing their level of involvement or adjustments in previously successful processes.
+
+# Planning Our Strategy
+
+We will use the OpenChain self-certification questionnaire as a way to check our current status versus the status required for OpenChain ISO/IEC 5230:2020 self-certification. This involves understanding the:
+
+1. 	Current policy for managing decisions around open source (and other) software.
+2.	 Current processes being used for managing open source (and other) software.
+3.	 Current records showing the usage of this software over the previous year.
+
+# Executing Our Plan
+
+## Step 1
+
+Identify a key sponsor for the project. Ideally, this will be a senior executive (C-suite) with ability to drive the activities, provide repeated vocal support and assist with budget needs.
+
+Determine the scope of the initial work. This may be targeting a single product line, or a particular development team, a complete business unit or perhaps even the whole company at once. 
+
+Form a team to carry out investigations.If the initial scope is wide, the team will need to incorporate representatives from across the targeted organization.
+
+## Step 2
+
+Carry out a survey of products produced by the target organization. Talk to representatives in the development teams, sales teams and finance teams, to gain an understanding of what is under development, what is being actively sold, what is being maintained, and what is still selling and shipping, even though no-one is working on it anymore.
+
+For each product identified, also survey the open source used in the product (if any), and the processes and policies used for developing, building and shipping the product, in as far as they relate to open source usage. Note compliance tools in use, if any.
+
```

Comment:
"Survey" may be done using a scanning tool. It may be difficult to find all the open source without tools to scan.